src/site/content/en/reliable/http-cache/index.md

```diff
@@ -55,44 +80,76 @@ with specific
+  no need to re-download it.
+* [`Last-Modified`][last-modified].
+  This header serves the same purpose as `ETag`, but is less accurate.
 
 Some web servers have built-in support for setting those headers by default,
 while others leave the headers out entirely unless you explicitly configure
 them. The specific details of _how_ to configure headers varies greatly
-depending on which web server you use, and you should consult the relevant
+depending on which web server you use, and you should consult your server's
 documentation to get the most accurate details.
 
 To save you some searching, here are instructions on configuring a few popular
 web servers:
 
-+  [Express](https://expressjs.com/en/api.html#express.static), running
-    on Node.js
++  [Express](https://expressjs.com/en/api.html#express.static)
 +  [Apache](https://httpd.apache.org/docs/2.4/caching.html)
 +  [nginx](http://nginx.org/en/docs/http/ngx_http_headers_module.html)
 +  [Firebase Hosting](https://firebase.google.com/docs/hosting/full-config)
 +  [Netlify](https://www.netlify.com/blog/2017/02/23/better-living-through-caching/)
 
-Leaving out the Cache-Control response header does not disable HTTP caching!
-Instead, browsers
-[effectively guess](https://www.mnot.net/blog/2017/03/16/browser-caching#heuristic-freshness)
+Leaving out the `Cache-Control` response header does not disable HTTP caching!
+Instead, browsers [effectively
+guess](https://www.mnot.net/blog/2017/03/16/browser-caching#heuristic-freshness)
 what type of caching behavior makes the most sense for a given type of content.
 Chances are you want more control than that offers, so take the time to
 configure your response headers.
 
-## Which response header values should you use?
+## Which response header values should you use? {: #response-header-strategies }
 
 There are two important scenarios that you should cover when configuring your
 web server's response headers.
 
-### Long-lived caching for versioned URLs
+### Long-lived caching for versioned URLs {: #versioned-urls }
+
+{% Details %}
+  {% DetailsSummary 'h4' %}
+    How versioned URLs can help your caching strategy
+    Versioned URLs are a good practice because they make it easier to invalidate
+    cached responses. 
+  {% endDetailsSummary %}
+  Suppose your server instructs browsers to cache a CSS file
+  for 1 year (<code>Cache-Control: max-age=31536000</code>) but your designer just made an
+  emergency update that you need to roll out immediately. How do you notify browsers
+  to update the "stale" cached copy of the file?
+  You can't, at least not without changing the URL of the resource. After the
+  browser caches the response, the cached version is used until it's no longer
+  fresh, as determined by <code>max-age</code> or <code>expires</code>, or until it is evicted from the cache
+  for some other reason— for example, the user clearing their browser cache. As a
```

Comment:
That looks like an extra space after the `—` character.

Reply:
```suggestion
  for some other reason—for example, the user clearing their browser cache. As a
```